site/app/views/calendar/CalendarView.php

```diff
@@ -40,11 +40,20 @@ public function showCalendar(CalendarInfo $info, array $courses, bool $in_course
         //Create list of courses and their term and get color
         $formatted_courses = [];
         $unformatted_courses = [];
+        $display_names = [];
         //$course_colors = [];
         foreach ($courses as $course) {
             $course_string = sprintf("%s %s", $course->getTitle(), $course->getTerm());
             array_push($formatted_courses, $course_string);
         }
+        $courseWithName = [];
+        foreach ($courses as $index => $course) {
+            $nameString = $course->getDisplayName() ?
+                        sprintf("%s %s", $course->getDisplayName(), $course->getTerm()) :
+                        sprintf("%s %s", $course->getTitle(), $course->getTerm());
+
+            $courseWithName[] = ['course' => $formatted_courses[$index], 'name' => $nameString];
```

Comment:
It would be better to break this into multiple lines so future diffs are easier to read.

```suggestion
            $courseWithName[] = [
                'course' => $formatted_courses[$index],
                'name' => $nameString,
            ];
```